src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -20,24 +20,44 @@ namespace Microsoft.PowerShell
     /// </summary>
     internal static class UpdatesNotification
     {
-        private const string UpdateCheckOptOutEnvVar = "POWERSHELL_UPDATECHECK_OPTOUT";
-        private const string Last3ReleasesUri = "https://api.github.com/repos/PowerShell/PowerShell/releases?per_page=3";
-        private const string LatestReleaseUri = "https://api.github.com/repos/PowerShell/PowerShell/releases/latest";
-
-        private const string SentinelFileName = "_sentinel_";
-        private const string DoneFileNameTemplate = "sentinel-{0}-{1}-{2}.done";
-        private const string DoneFileNamePattern = "sentinel-*.done";
-        private const string UpdateFileNameTemplate = "update_{0}_{1}";
-        private const string UpdateFileNamePattern = "update_v*.*.*_????-??-??";
-
-        private static readonly EnumerationOptions s_enumOptions = new EnumerationOptions();
-        private static readonly string s_cacheDirectory = Path.Combine(Platform.CacheDirectory, PSVersionInfo.GitCommitId);
+        private const string UpdateCheckEnvVar = "POWERSHELL_UPDATECHECK";
+        private const string LTSBuildInfoURL = "https://pscoretestdata.blob.core.windows.net/buildinfo/lts.json";
+        private const string StableBuildInfoURL = "https://pscoretestdata.blob.core.windows.net/buildinfo/stable.json";
+        private const string PreviewBuildInfoURL = "https://pscoretestdata.blob.core.windows.net/buildinfo/preview.json";
```

Comment:
Thinking about LTS version maybe move these consts in a powershell config file?

Reply:
Why? Those URLs are not configurable.

Reply:
The URLs could be changed in next three years. So we could either move the URLs to a config or use official and stable aka.ms URLs.

Reply:
Good point about the aka.ms URLs. I will change to the aka.ms forward links.

Reply:
Third option that may not be feasible with current infrastructure, but is something I've seen in other update infrastructure, is something like `aka.ms/updatepowershell?channel={channel}` where you can configure the channel.

One of the nicest examples of this is VSCode's update blob URL, which is `https://vscode-update.azurewebsites.net/$version/$platform/$channel`. We use that in [the `install-vscode.ps1` script](https://github.com/PowerShell/vscode-powershell/blob/0a5b5be7235414924bc0636b07b3bf382108a801/scripts/Install-VSCode.ps1#L378).

Reply:
Yes, I like a single endpoint for all the needed information, but what you mentioned above would need a real endpoint, not just a blob storage.
I talked with @TravisEz13 offline about having a single .json file that contains latest information about all channels. But such a single .json file cannot be updated by individual builds because we can have multiple release builds at the same time (ex. a LTS servicing release and a new stable release at the same time), so we will still need individual .json files for the release of each channel.

Reply:
I pushed a commit to address @rjmholt's comment, but haven't updated to use the `aka.ms` link yet. Need to talk to @TravisEz13 on what name to use for the link when he is back.

Reply:
Update to use the `aka.ms` links instead.